utils/comment.go

```diff
@@ -201,6 +202,12 @@ func getNewReviewComments(repo *Repository, issues *IssuesCollection) (commentsT
 	for _, sast := range issues.Sast {
 		commentsToAdd = append(commentsToAdd, generateReviewComment(SastComment, sast.Location, generateSourceCodeReviewContent(SastComment, sast, writer)))
 	}
+	if !repo.Params.PullRequestSecretComments {
+		return
+	}
+	for _, secret := range issues.Secrets {
```

Comment:
I saw you changed in one of your recent PRs to iterate over unique issues (iac, sast - GetUniqueIacIssues, GetUniqueSastIssues)
Should we do the same here?

Reply:
violation not exists here